cwrap/clib.py

```diff
@@ -42,7 +42,7 @@
                 "darwin" : "dylib" }
 
 
-# Passing None to the CDLL() function means to open a lib handle to
+# Passing None to the PyDLL() function means to open a lib handle to
 # the current runnning process, i.e. like dlopen( NULL ). We must
 # special case this to avoid creating the bogus argument 'None.so'.
```

Comment:
Also, I have no idea what this comment means. Passing None to CDLL won't "create the bogus argument 'None.so'".